packages/ui/src/ui/utils/acl/index.ts

```diff
@@ -17,9 +19,9 @@ const POOL_TREES_REGEXP = new RegExp('//sys/pool_trees/[^/]+/.+');
 const TCB_REGEXP = new RegExp('//sys/tablet_cell_bundles/.+');
 
 // //sys/access_control_object_namespaces...
-const ACCESS_CONTROL_OBJECT_ANY_REGEXP = new RegExp(
-    '//sys/access_control_object_namespaces[^/+]{0,}',
-);
+const ACCESS_CONTROL_OBJECT_ANY_REGEXP =
```

Comment:
```ts
const ACCESS_CONTROL_OBJECT_ANY_REGEXP = makeRegexpFromSettings(uiSettings.reUseEffectiveAclForPath);
```

Reply:
According to current implementation of `makeRegexpFromSettings` you should get `undefined`

Reply:
Oh, sorry, I am wrong. Can we rewrite `makeRegexpFromSettings` to have `undefined` for the case?